### DISCUSSION TITLE
when I use "contain: content", how can I change to tailwind css?

### DISCUSSION THREAD

**Discussion by tobistudio** [tailwindlabs/tailwindcss#12413] [Help]
I gonna use "contain: content" style in tailwind css as classname, By the way, I can't find this way now.
Please help me about this problem.

**Comment by wongjn** [comment]
You could consider:
- Using an [arbitrary property class](https://tailwindcss.com/docs/adding-custom-styles#arbitrary-properties) like `[contain:content]`.
- [Creating/using a Tailwind plugin](https://tailwindcss.com/docs/plugins#adding-utilities) that adds a class that applies `contain: content`.

**Comment by cupid20103** [comment]
Use [contain:content] in inline CSS definitions.
